CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.4 
+
+* Alter default host on devevopment mode [#1369](https://github.com/sinatra/sinatra/issues/1369) by David Silveira
```

Comment:
Just a small thing, development has a typo.

Reply:
I think it would be good to the describe the change too, so the reader doesn't have to open up the issue